clients/src/main/java/org/apache/kafka/server/quota/ClientQuotaCallback.java

```diff
@@ -1,4 +1,5 @@
 /*
```

Comment:
The file has increased in 1 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._